server/backend/influxdb.go

```diff
@@ -0,0 +1,39 @@
+package backend
+
+import (
+	"context"
+	"errors"
+	"github.com/TUM-Dev/Campus-Backend/server/backend/influx"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"os"
+)
+
+const (
+	// InfluxBatchSize is the number of points that are sent to influxdb at once
+	// This is to reduce the number of requests to the database
+	InfluxBatchSize = 1_000
+)
+
+var (
+	influxToken = os.Getenv("INFLUXDB_TOKEN")
+	influxURL   = os.Getenv("INFLUXDB_URL")
+)
+
+func ConnectToInfluxDB() error {
```

Comment:
Seems like all this could go into the influx package, no?

Reply:
I think I can't access the influx package from the main file because it is in the backend package.